JavaDirectory/src/main/java/com/bug/free/invention/api/ApiApplication.java

```diff
@@ -7,7 +7,11 @@
 public class ApiApplication {
 
 	public static void main(String[] args) {
-		SpringApplication.run(ApiApplication.class, args);
-	}
+		try {
+			SpringApplication.run(ApiApplication.class, args);
+		} catch (Exception E) {
+			E.printStackTrace();
+		}
```

Comment:
Variables should not be Capitalised
```suggestion
		} catch (Exception e) {
			e.printStackTrace();
		}
```